docs/overview/api-keys.md

```diff
@@ -53,4 +53,25 @@ var map = L.Mapzen.map('map', {
 }
 ```
 
+Another method of obfuscating the API key is put it in a separate YAML file and use an `import:` statement to link to it from your main scene file. In Tangram Play, this would look like:
+
+```
+import:
+    https://your-file-url.yaml #link to your YAML file
+
+sources:
+    mapzen:
+      [...]
```

Comment:
tiny formatting nitpick: the second and last lines in this block use an indent of 4, but these middle two use 2, and our yaml parser requires that a consistent number of spaces throughout the file. so we might true these up just in case somebody copies-and-pastes this example